.github/workflows/test-desktop-reusable.yml

```diff
@@ -82,10 +82,11 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref || github.sha }}
-      - name: Setup the toolchain
-        id: toolchain
-        uses: LedgerHQ/ledger-live/tools/actions/composites/setup-toolchain@develop
+      - name: Setup the caches
```

Comment:
question: do we really need another setup cache here as it's already done in line 43 ? am i wrong ?

Reply:
this is a different job - you can see if you expand the collapsed code between the 2 invocations